### PR TITLE
feat(platform): support containerd on installer

### PR DESCRIPTION
**Pull request by Huimintai**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: build/docker/tools/tke-installer/Dockerfile [hunk 1]
```diff
@@ -14,14 +14,15 @@
 # WARRANTIES OF ANY KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations under the License.
 
-FROM alpine:3.10
+FROM docker:dind
```
Comment (leoryu) [inline build/docker/tools/tke-installer/Dockerfile:17]: replace tag to 20.10.8-dind
dind tag will always pull latest image

File: build/docker/tools/tke-installer/run.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
Comment (leoryu) [inline build/docker/tools/tke-installer/run.sh:1]: add license to shell

File: build/docker/tools/tke-installer/Dockerfile (hunk 1 above)
Comment (Huimintai) [reply build/docker/tools/tke-installer/Dockerfile:17]: Done.

File: build/docker/tools/tke-installer/dockerd-entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,191 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (Huimintai) [reply build/docker/tools/tke-installer/dockerd-entrypoint.sh:1]: Done.

File: build/docker/tools/tke-installer/run.sh (hunk 2 above)
Comment (Huimintai) [reply build/docker/tools/tke-installer/run.sh:1]: Done.

**Pull request merged**